### PR TITLE
Improve instructions

### PR DESCRIPTION
**Pull request by 4812571**
I think I've improved the instructions and removed a number of unclear points! They're not perfect yet, but these should be an improvement to some onboarding issues I had myself.

### CODE REVIEW

File: gh-actions.md [hunk 1]
```diff
@@ -53,17 +53,27 @@ In the CMake and libFuzzer exercises, we forked the original or "upstream" mayhe
     cp Dockerfile.template mayhem/Dockerfile
     ```
 
-2. There are two comments marked as TODO in the Dockerfile you'll need to change in your favorite text editor. For the first section, you need to add the build commands from steps 6-8 of the previous section using Dockerfile `RUN` entries. Under the second TODO, you need to change the COPY command so that it copies the libFuzzer executable (that you ran in step 9 of the previous section) into the packaging stage.
+2. There are two comments marked TODO in the Dockerfile you'll need to change in your favorite text editor. For the first comment, you will need to add Dockerfile `RUN` entires to run the build commands from the previous exercise.
```
Comment (xansec) [inline gh-actions.md:56]: Hmm, I actually don't want to give too much away here. A solution branch exists where people can see the correct Dockerfile commands anyways, so I don't want to also include it here.

Comment (xansec) [reply gh-actions.md:56]: https://github.com/mayhemheroes/mayhem-cmake-example/tree/solution

File: README.md [hunk 2]
```diff
@@ -14,10 +14,10 @@
 
 ## Exercises
 
-1. [lighttpd exercise](lighttpd-example.md)
+1. [lighttpd Exercise](lighttpd-example.md)
 2. [Docker + Mayhem Exercise](docker-intro.md)
-3. [CMake Exercise](cmake-libfuzzer-exercise.md#cmake-exercise)
```
Comment (xansec) [inline README.md:19]: It looks like you want to split up the cmake and libfuzzer exercises. I can do this directly.

File: gh-actions.md (hunk 1 above)
Comment (4812571) [reply gh-actions.md:56]: I see! I wasn't aware there was a solution branch! I didn't think to look.

File: README.md (hunk 2 above)
Comment (4812571) [reply README.md:19]: That's true. I'll mark this pull request as closed and allow you to do that directly.

**Pull request closed without merging**